From c6ef874ba4a7fa06a4ae7bde5f527bf0fb2e203f Mon Sep 17 00:00:00 2001
From: Huynh Thanh Hung <hung.huynh,wz@renesas.com>
Date: Tue, 28 May 2019 11:16:50 +0700
Subject: [PATCH 163/248] arm64: dts: renesas: hihope-common: enable usb3_phy0
 node

This patch enables usb3_phy0 node and set reference clock usb3s0_clk.

Signed-off-by: Huynh Thanh Hung <hung.huynh,wz@renesas.com>
---
 arch/arm64/boot/dts/renesas/hihope-common.dtsi | 8 ++++++++
 1 file changed, 8 insertions(+)

diff --git a/arch/arm64/boot/dts/renesas/hihope-common.dtsi b/arch/arm64/boot/dts/renesas/hihope-common.dtsi
index 28a55b4..86d5403 100644
--- a/arch/arm64/boot/dts/renesas/hihope-common.dtsi
+++ b/arch/arm64/boot/dts/renesas/hihope-common.dtsi
@@ -88,6 +88,10 @@
 	clock-frequency = <50000000>;
 };
 
+&usb3s0_clk {
+	clock-frequency = <100000000>;
+};
+
 &pfc {
 	pinctrl-0 = <&scif_clk_pins>;
 	pinctrl-names = "default";
@@ -284,6 +288,10 @@
 	status = "okay";
 };
 
+&usb3_phy0 {
+	status = "okay";
+};
+
 &hsusb {
 	dr_mode = "otg";
 	status = "okay";
-- 
2.7.4

